tsconfig.json

```diff
@@ -0,0 +1,33 @@
+{
+  "compilerOptions": {
+    /**
+     * Must allow:
+     */
+    "downlevelIteration": true,
+    "resolveJsonModule": true,
+    "experimentalDecorators": true,
+    "noUnusedParameters": false,
+    "removeComments": false,
+    "incremental": true,
+
+    /**
+     * Nice to have:
+     */
+    // "preserveConstEnums": true,
+    // "declaration": true,
+    // "allowUnreachableCode": false,
+    // "pretty": true,
+
```

Comment:
I agree to make `preserveConstEnums` true. It make generated code more readable, rather than just number/string without reference to the enum names.
All the other options are just default values, so they can be safely removed.